app/controllers/suggest_users_controller.rb

```diff
@@ -1,3 +1,22 @@
 class SuggestUsersController < ApplicationController
-  def show; end
+  before_action :load_passive_user, except: :show
+
+  def show
+    @user = User.get_suggest_user(current_user).first
+  end
+
+  def react
+    current_user.send(@react.to_sym, @pass_user)
+    @user_next = User.get_suggest_user(current_user).first
+    respond_to :js
+  end
+
+  private
+
+  def load_passive_user
+    if params[:passive_user_id].present? && params[:react].present?
```

Comment:
```suggestion
    params.values_at(:passive_user_id, :react).all?(&:present?)
```
- Dùng unless thay cho if; end

Reply:
vâng a